src/main/java/me/totalfreedom/totalfreedommod/admin/AdminList.java

```diff
@@ -167,36 +167,19 @@ public Admin getAdmin(Player player)
         String ip = Ips.getIp(player);
         Admin admin = getEntryByName(player.getName());
 
-        // Admin by name
-        if (admin != null)
+        if (admin != null && admin.getName().equalsIgnoreCase(player.getName()))
```

Comment:
why do you need to check if name is same? getEntryByName only gets if name is same?

Reply:
Ah my bad. I’ll make some changes when I get home.